### PR TITLE
Add FSI section to Input File docs.

### PR DESCRIPTION
**Pull request by neilmatula**

### CODE REVIEW

File: docs/source/user/nalu_run/nalu_inp.rst [hunk 1]
```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
...
+   ``t_end`` divided by ``dt_FAST`` from the OpenFAST standalone run.
+
+.. inpfile:: turbine_name.sim_type
+
+   String (in double quotes) indicating the type of OpenFAST simulation.
+   For FSI simulations, this should always be set to ``"ext-loads"``.
+
+.. inpfile:: turbine_name.blade_parts
+
+   List of strings (in double quotes) indicating the mesh element blocks
```
Comment (psakievich) [inline docs/source/user/nalu_run/nalu_inp.rst:1274]: Double quotes are not required for yaml to recognize strings.  Raw characters are fine.

File: docs/source/user/nalu_run/nalu_inp.rst [hunk 2]
```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
...
+.. inpfile:: turbine_name.tower_boundary_parts
+
+   List of strings (in double quotes) indicating the mesh sidesets
+   corresponding to the wall boundaries of the tower.
+
+.. inpfile:: turbine_name.az_blend_mean
+
+   Real variable (in radians) indicating the mean angular position for the load blending.
+   The loads provided to BeamDyn are a weighted average of the loads
+   provided by AeroDyn and the true CFD loads provided by Nalu-Wind.
```
Comment (psakievich) [inline docs/source/user/nalu_run/nalu_inp.rst:1296]:
```suggestion
   provided by AeroDyn and the true CFD loads provided by Nalu-Wind during startup to help with stability while the CFD flow field develops around the turbine structure.
```

File: docs/source/user/nalu_run/nalu_inp.rst [hunk 3]
```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
...
+   The weight of the Nalu-Wind contribution takes the form
+   :math:`\frac{1}{2} \left[1+\text{tanh}\left(\left(\phi-\phi_\text{mean}\right)/\phi_\text{delta}\right)\right]`,
+   where :math:`\phi` is the angle that the turbine has rotated through,
```
Comment (psakievich) [inline docs/source/user/nalu_run/nalu_inp.rst:1299]:
```suggestion
   where :math:`\phi` is the angle that the turbine has rotated through (including any initial OpenFAST runs),
```

File: docs/source/user/nalu_run/nalu_inp.rst [hunk 4]
```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
...
+.. inpfile:: turbine_name.az_blend_delta
+
+   Real variable (in radians) indicating the width for load blending.
+   The loads provided to BeamDyn are a weighted average of the loads
+   provided by AeroDyn and the true CFD loads provided by Nalu-Wind.
+   The weight of the Nalu-Wind contribution takes the form
+   :math:`\frac{1}{2} \left[1+\text{tanh}\left(\left(\phi-\phi_\text{mean}\right)/\phi_\text{delta}\right)\right]`,
+   where :math:`\phi` is the angle that the turbine has rotated through,
+   :math:`\phi_\text{mean}` is the value of ``az_blend_mean``, and
+   :math:`\phi_\text{delta}` is the value of ``az_blend_delta``.
```
Comment (psakievich) [inline docs/source/user/nalu_run/nalu_inp.rst:1312]:
```suggestion
Definition supplied in the `turbine_name.az_blend_mean` parameter's description.
```
No need to repeat ourselves here 😎

File: docs/source/user/nalu_run/nalu_inp.rst [hunk 5]
```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
...
+.. inpfile:: turbine_name.shear_exp
+
+   Real variable indicating the exponent used in a power-law approximation
+   of the incoming ABL.  This variable should match the variable with the same
+   name in the OpenFAST driver input file (with extension .yaml).
+
+.. inpfile:: turbine_name.deflection_ramping
+
+   The ``deflection_ramping`` sub-subsection controls the temporal and spatial
+   ramping of blade deflections applied in Nalu.  The temporal ramping enables a smooth
```
Comment (psakievich) [inline docs/source/user/nalu_run/nalu_inp.rst:1346]:
```suggestion
   ramping of blade deflections applied in Nalu-Wind.  The temporal ramping enables a smooth
```

File: docs/source/user/nalu_run/nalu_inp.rst [hunk 6]
```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
...
+   transition from the reference configuration to the full deflections, and
```
Comment (psakievich) [inline docs/source/user/nalu_run/nalu_inp.rst:1347]:
```suggestion
   transition for the blades from a rigid body motion based on the hub motion to the full blade deflections, and
```

File: docs/source/user/nalu_run/nalu_inp.rst [hunk 7]
```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
...
+   improves the stability of the simulation.  The span ramping enables the root section
```
Comment (psakievich) [inline docs/source/user/nalu_run/nalu_inp.rst:1348]:
```suggestion
   improves the stability of the simulation during startup.  The span ramping enables the root section
```

File: docs/source/user/nalu_run/nalu_inp.rst [hunk 8]
```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
...
+.. inpfile:: deflection_ramping.span_ramp_distance
+
+   Real variable indicating the distance from the root over which to ramp deflections
+   if ``enable_span_ramping`` is set to true.  Distance is measured in the same units
+   as the mesh (meters in most cases).
+
+.. inpfile:: deflection_ramping.temporal_ramp_start
+
+   Real variable indicating the time at which temporal ramping should begin.
+   Prior to this time, zero deflections will be provided.  Note that this
```
Comment (psakievich) [inline docs/source/user/nalu_run/nalu_inp.rst:1377]:
```suggestion
   Prior to this time, zero blade deflections will be provided.  Note that this
```

File: docs/source/user/nalu_run/nalu_inp.rst [hunk 9]
```diff
@@ -1139,6 +1139,275 @@ Time-step Control Options
...
+   should correspond to time as recorded by Nalu-Wind and described in the
+   ``Time_Integrators`` section, not as recorded by OpenFAST.  Hence, this will
+   not correspond to ``t_start`` above.
+
+.. inpfile:: deflection_ramping.temporal_ramp_end
+
+   Real variable indicating the time at which temporal ramping should end.
+   After this time, full deflections will be provided.  Note that this
```
Comment (psakievich) [inline docs/source/user/nalu_run/nalu_inp.rst:1385]:
```suggestion
   After this time, full blade deflections will be provided.  Note that this
```

**Review by psakievich — CHANGES_REQUESTED** [review]
Looks good. Thanks for typing all this up. Requesting a few edits.

**Comment by psakievich** [comment]
Middle equation, I believe it should be `\theta-\theta_0` in the numerator.

**Comment by neilmatula** [comment]
@psakievich 
Theta is measured from the top blade in that figure, right?
If it's theta-theta0 in the blue (ramped) region, that would be a negative number.
Unless I've got it totally wrong

**Comment by psakievich** [comment]
> @psakievich Theta is measured from the top blade in that figure, right? If it's theta-theta0 in the blue (ramped) region, that would be a negative number. Unless I've got it totally wrong

Ah yeah you are right.

**Review by psakievich — APPROVED** [review]

**Pull request merged**